tests/test_AdType.py

```diff
@@ -0,0 +1,13 @@
+from src.AdType import AdType
+
+
+def test_AdType():
+    at1 = AdType(ad_type_id='0011',
+                 ad_type_name='Chicago Bears',
+                 cost_share_rate=0.50,
+                 allowed_spend_per_ad=range(200, 201))
```

Comment:
this is wrong, this is split into two different parameters now.
please fix.

Reply:
there actually is not even an AdType needed anymore, so this test can be deleted.